2024/planning-process.md

```diff
@@ -0,0 +1,132 @@
+4. Focus Area proposals must:
+    1. Name a specific web technology.
+        1. This can be a web technology that has not yet been implemented in any or all browser engines.
+        2. Or this can be a web technology that has already been implemented in all three engines, but those implementations have test failures that reveal a lack of compliance to the web standard, causing a lack of interoperability.
+    2. The feature must be defined in a sufficiently mature standards-track web specification from one the following organizations. Please provide link to the web standard.
+        1. [IETF](https://www.ietf.org/) (Standard Track)
+        2. [Khronos Group](https://www.khronos.org/)
+        3. [TC39](https://tc39.es/) (Stage 2+)
+        4. [W3C](https://www.w3.org/) ([Recommendation Track](https://www.w3.org/2023/Process-20230612/#rec-track))
+        5. [WHATWG](https://whatwg.org/)
+    3. Describe the benefit to web developers and the web itself
+    4. Link to existing tests (if any) and/or describe plan to create needed tests
+5. Investigation Project proposals must:
+    1. Outline their scope and their goals,
+    2. Not substantially be work better suited to a group within a standards development organization,
+    3. Clearly provide present or future impact on cross-browser interoperability.
+
+As soon as each proposal is submitted as a GitHub issue, discussion of that proposal can begin. Questions can be asked. The proposal can be edited and refined.
+
+At the end of the three weeks, the deadline will end the period when brand new proposals can be submitted.
+
+A fourth week will allow for continued discussion and refinement of already-submitted proposals. The deadline at the end of the fourth week ends the opportunity to significantly change the scope of proposals.  The proposals will be evaluated on their state at that time.
+
+
+## Focus Area Selection
+
+### Elimination Round 1: Disqualification
+
+After the Call of Proposals Period has ended, the Interop Team will finalize an official list of all Focus Area proposals.
+
+Any Focus Area proposal that does not meet the minimum publicly-stated selection criteria will be considered for preliminary elimination. For example:
+
+1. The proposal is overly vague or broad.
+2. The proposed technology is already interoperable according to WPT results.
+3. The proposed technology is not on a standards track, or its standard is not sufficiently mature.
+
+Decisions will be made at a specific meeting for discussion of preliminary eliminations. The minutes of this meeting are confidential to the team. Any member organization can bring up a proposal they believe should be eliminated. Brief debate can be had. If **there is quick consensus**, that proposal will be eliminated. If there is not consensus and a longer debate is needed, the proposal will be not be eliminated and will instead move on to the next round.
```

Comment:
I'm not convinced that the phrasing that it's specifically the "minutes of the meeting" which are confidential captures the intent. AIUI the intent is that vendor positions are confidential throughout, irrespective of how they're shared (e.g. if the data is captured in a spreadsheet outside of the context of a specific meeting).

Although the document does also say that those minutes are shared in a specific meeting, in practice I think it's likely that we'll want some kind of async method to collate and share information.

Reply:
Drafting up this instead (in the GDoc):
> The positions of individual member organizations are confidential to the team.